lib/ledger-node-operations.js

```diff
@@ -43,15 +71,35 @@ module.exports = class LedgerNodeOperations {
    *
    * operation - the operation to validate.
    * callback(err) - the callback to call when finished.
-   *   err - An Error if an error occurred, null otherwise.
+   *   err - A BedrockError if an error occurred, null otherwise.
    */
   validate(operation, callback) {
```

Comment:
Does it make sense to use the `{operation}` style here?

Reply:
It does make sense, but I see this being more than just a style change, it would be a breaking change. @mattcollier @dlongley

Reply:
@gannan08,

We need to break it and fix all the things before we do a release. :)

We want consistent style.

Reply:
I believe this may be the only place is API is used outside of this module, please put a PR in there as well: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/master/lib/events.js#L1305